sdk/src/introspection.ts

```diff
@@ -16,3 +20,35 @@ export function getInterfaceId(iface: Interface): string {
   }
   return interfaceId.toHexString();
 }
+
+/**
+ * Gets the protocol version of a contract, if the contract does not have a
+ * protocolVersion function, it will return [1, 0, 0]
+ *
+ * @export
+ * @param {string} rpc
+ * @param {string} contractAddress
+ * @return {*}  {Promise<[number, number, number]>}
+ */
+export async function getProtocolVersion(
+  rpc: string,
+  contractAddress: string
+): Promise<[number, number, number]> {
+  if (!isAddress(contractAddress)) {
+    throw new InvalidAddressError(contractAddress);
+  }
+  const provider = new JsonRpcProvider(rpc);
+  const iface = new Interface([
+    'function protocolVersion() public pure returns (uint8[3] memory)',
+  ]);
+  const contract = new Contract(contractAddress, iface, provider);
+  let version: [number, number, number];
+  try {
+    version = await contract.protocolVersion();
+  } catch (e) {
+    // ethers5 throws a call exception error which could mean a lot of things
+    // so this is not accurate
+    version = [1, 0, 0];
```

Comment:
is it still in the sdk a constants file? could we move this there? 
magic numbers could be confusing

Reply:
nope, we dont have a constants file anymore, but i dont see it as necessary it's only used there and it does not add, that much value